hack/deploy.sh

```diff
@@ -5,7 +5,8 @@ set -euxo pipefail
 source "$(dirname $0)/common"
 
 if [ $REMOTE_REGISTRY = false ] ; then
-    oc create -n ${NAMESPACE} -f manifests/05-deployment.yaml
+    $repo_dir/hack/gen-olm-artifacts.sh ${CSV_FILE} ${NAMESPACE} 'dep' \
```

Comment:
can you explain what this line is doing? why is there a `dep` call?

Reply:
possibly bad naming... it short hand for generate the kind 'deployment'